tests/framework/microvms.py

```diff
@@ -126,4 +133,6 @@ def spawn(cls, bin_cloner_path, start=False,
                                      None,
                                      start,
                                      fc_binary,
-                                     jailer_binary)
+                                     jailer_binary,
+                                     net_ifaces=net_ifaces,
+                                     enable_diff_snapshots=diff_snapshots)
```

Comment:
Nit here and on lines 117-118:
To keep the style, you could pass directly **net_ifaces** instead of **net_ifaces=net_ifaces**, same with diff_snapshots :)

Reply:
When multiple parameters are passed, it's easier to understand what parameters are passed when mentioning each one by key word. 
I'd rather go the other way around and pass each parameter explicitly.